container/container_contract.go

```diff
@@ -62,6 +62,8 @@ const (
 
 	singleEstimatePrefix = "est"
 	estimateKeyPrefix    = "cnr"
+	containerKeyPrefix   = "cid"
+	ownerKeyPrefix       = "own"
```

Comment:
We have some 3-letter prefixes already unfortunately, but I'd use 1-byte for new ones anyway.

Reply:
Fixed. Initially I wanted to use 'c' as a prefix which would conflict with `cnr`, but nobody cares really.